src/models/userDao.js

```diff
@@ -0,0 +1,44 @@
+const { database } = require("./dataSource");
+
+const createUser = async (
+  userId,
+  nickname,
+  gender,
+  hashedPw,
+  profileImg,
+  age
+) => {
+  const user = await database.query(
+    `SELECT 
+            *
+        FROM users
+        WHERE users.user_id = ?
+        `,
+    [userId]
+  );
+
+  if (userId === user.userId) {
+    const error = new Error(`${userId} is already exist`);
+    error.statusCode = 400;
+    throw error;
+  }
+
+  const newUser = await database.query(
```

Comment:
```suggestion
  const result = await database.query(
```
dao에서 회원가입에 대한 함수를 createUser라고 지어주셨기 때문에 중복을 피하기 위해 database.query에 대한 코드를 newUser라는 변수에 담아주신 것 같네요! 이런경우에는 result라고 변수명을 짓는 방법도 있습니다.